config/test.cjs

```diff
@@ -7,6 +7,11 @@ module.exports = {
 	admin: {
 		key: 'admin',
 	},
+	geoip: {
+		cache: {
+			ttl: 1, // 1 ms ttl here to disable redis cache in tests
```

Comment:
> Additionally redis cached some of the geo ip values during tests, which made them pass locally and fail on CI.

Do you mean from previous tests or from dev runs? Because this will fix test mixups, but if something is already cached in redis, it'll stay there (unless we delete it which I don't think we do). In other projects, I solve this by calling flushDb() in dev/test envs on startup.

Reply:
Both, `ttl: 1` is to fix previous test mixups. And [fakeIpForDebug](https://github.com/jsdelivr/globalping/pull/375/files#diff-24fca2bd712e62250531a7ec03d88f45b2874994059c608fda0712b1f534f87eR25) update is to fix prev dev runs cache. flushDb seems to be more elegant, so I will replace the second update with it.

Reply:
👍 here for reference: https://github.com/jsdelivr/data.jsdelivr.com/blob/master/src/lib/redis/index.js#L10-L12